src/js/components/FilterInputText.js

```diff
@@ -92,15 +92,15 @@ class FilterInputText extends React.Component {
     });
 
     return (
-      <span className="form-control-group-add-on form-control-group-add-on-append">
+      <span className="form-control-group-add-on">
```

Comment:
The `FormsTabContent` component is part of @ashenden style guide; I suspect it's just leftover from the previous Canvas documentation. The `form-control-group-add-on*` family of classes should be styled via CNVS, not via some custom styling we added to reactjs-components, in my opinion.

In any case, the `form-control-group-add-on-append` class in reactjs-components has no properties that are being applied here, so it should be removed even if we decide to keep the custom styles in reactjs-components.